drivers/mpu6050/MPU6050.cpp

```diff
@@ -138,7 +138,7 @@ struct fifo_packet {
 
 // Length of the FIFO used by the sensor to buffer unread
 // sensor data.
-#define MPU_MAX_LEN_FIFO_IN_BYTES 1024
+#define MPU_MAX_LEN_FIFO_IN_BYTES (4 * sizeof(fifo_packet))
```

Comment:
In general, I wanted to reduce allocated memory. We sample at 1kHz and read the data at 1kHz. Thus, we most often read just one `fifo_packet` with a few edge cases, where we have no packet ready in this read cycle and two in the next cycle. I then wanted to give it some slack and doubled the maximum size.
Do you think this makes sense? I then would add a comment in code. Or should I undo the changes?